agb/examples/affine_object.rs

```diff
@@ -0,0 +1,70 @@
+#![no_std]
+#![no_main]
+
+use agb::{
+    display::{
+        GraphicsFrame, HEIGHT, Rgb15, WIDTH,
+        affine::AffineMatrix,
+        object::{AffineMatrixInstance, AffineMode, Object, ObjectAffine, SpriteVram},
+        tiled::VRAM_MANAGER,
+    },
+    fixnum::{Num, num, vec2},
+    include_aseprite,
+};
+
+include_aseprite!(mod sprites, "examples/gfx/crab.aseprite", "examples/gfx/box.aseprite");
+
+fn x(idx: i32) -> i32 {
+    (WIDTH / 4) * (idx + 1) - 16
+}
+
+fn show_with_boxes(matrix: AffineMatrix, height: i32, frame: &mut GraphicsFrame) {
+    let crab = SpriteVram::from(sprites::IDLE.sprite(0));
+    let square = SpriteVram::from(sprites::BOX.sprite(0));
+    let instance = AffineMatrixInstance::new(matrix.to_object_wrapping());
+
+    for idx in 0..3 {
+        Object::new(square.clone())
+            .set_position((x(idx), height))
+            .show(frame);
+    }
+
+    Object::new(crab.clone())
+        .set_position((x(0), height))
+        .show(frame);
+    ObjectAffine::new(crab.clone(), instance.clone(), AffineMode::Affine)
+        .set_position((x(1), height))
+        .show(frame);
+    ObjectAffine::new(crab.clone(), instance.clone(), AffineMode::AffineDouble)
+        .set_position((x(2), height))
+        .show(frame);
+}
+
+#[agb::entry]
+fn main(mut gba: agb::Gba) -> ! {
+    let mut gfx = gba.graphics.get();
+
+    VRAM_MANAGER.set_background_palette_colour(0, 0, Rgb15::WHITE);
```

Comment:
This is a nice way to just change the background colour, we should maybe do that for other ones. I can take a look when I do the audit of all the examples